src/main/java/com/moodybluez/enterprise/service/MoodService.java

```diff
@@ -33,8 +33,7 @@ public void delete(int moodId) throws Exception {
     @Override
     @Cacheable(value="mood", key="#id")
     public Mood fetchById(int moodId) {
-        Mood foundMood = moodDAO.fetchByID(moodId);
-        return foundMood;
+        return moodDAO.fetchByID(moodId);
```

Comment:
Ditto - "No need to define variable and return, just return what the method returns if you don't need to modify it."